app/views/api/v1/posts/show.json.jbuilder

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-json.call(@post, :id, :title, :body, :published_at, :user_id)
+json.partial! 'post_info', post: @post
 
 json.likes @post.likes do |like|
   json.user_id like.user_id
```

Comment:
let's include everything in the partial, even though it will be empty on the creation